src/reducers/notificationReducer.js

```diff
@@ -0,0 +1,18 @@
+const notificationReducer = (state, action) => {
+  switch (action.type) {
+    case 'ADD_NOTIFICATION':
+      return {
+        ...state,
+        notifications: [...state.notifications, action.payload]
+      };
+    case 'DELETE_NOTIFICATION':
+      const updatedNotifications = state.notifications.filter(
```

Comment:
Lint error could be removed by moving `state.notifications.filter((notification) => notification.id !== action.payload)` to inside the return call without declaring a new `const` inside the switch case.

https://github.com/codeforpdx/PASS/assets/14917816/1d62ed4b-12a7-49c4-85ab-d261a24941c2

Reply:
@leekahung Thanks Ka Hung. That helped me get all errors fixed.